routes/event_details.js

```diff
@@ -0,0 +1,19 @@
+var request = require('request');
```

Comment:
you should use `const` instead of `var` in this file, as you are using it in your other files on the server.